pen/static/pen/css/style.css

```diff
@@ -0,0 +1,79 @@
+.highlight
+{
+    border: 3px solid black;
+    font-weight: bold;
+    background-color: red;
+    height: 70px;
+    width: 200px;
+    font-size: 45px;
+    text-align: center;
+    border-radius: 50px;
+    border: none;
```

Comment:
property border się dubluje. Do wybrania jeden, a drugi do usunięcia.

Reply:
Line no. 11 of the code has been removed